github/ci/prow/files/jobs/project-infra/project-infra-postsubmits.yaml

```diff
@@ -8,7 +8,7 @@ postsubmits:
       labels:
         preset-dind-enabled: "true"
         preset-docker-mirror: "true"
-        preset-kubevirtci-docker-credential: "true"
+        preset-project-infra-kubevirtci-docker-credentia: "true"
```

Comment:
I think it is missing 'l' at the end of the name?

Reply:
fixed